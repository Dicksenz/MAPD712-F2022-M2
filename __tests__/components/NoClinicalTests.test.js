import React from "react";
import renderer from "react-test-renderer";
import NoClinicalTests from "../../components/NoClincalTests";

test("Check if NoClinicalTests component renders correctly", () => {
  const tree = renderer.create(<NoClinicalTests />).toJSON();
  expect(tree).toMatchSnapshot();
});
