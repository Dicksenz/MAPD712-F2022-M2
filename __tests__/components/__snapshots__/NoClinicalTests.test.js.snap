// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if NoClinicalTests component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    No clinical tests found
  </Text>
</View>
`;
